v2/pkg/constants/constants.go

```diff
@@ -10,13 +10,19 @@ const (
 const (
 	LabelPool     = "coil.cybozu.com/pool"
 	LabelNode     = "coil.cybozu.com/node"
+	LabelRequest  = "coil.cybozu.com/request"
 	LabelReserved = "coil.cybozu.com/reserved"
 
 	LabelAppName      = "app.kubernetes.io/name"
 	LabelAppInstance  = "app.kubernetes.io/instance"
 	LabelAppComponent = "app.kubernetes.io/component"
 )
 
+// Index keys
+const (
+	AddressBlockRequestKey = "address-block.request"
```

Comment:
This could be changed to `AddressBlockRequestLabel = "coil.cybozu.com/request"` or something similar to be a bit more in line with k8s, but it's not that important.